server.go

```diff
@@ -168,6 +171,8 @@ func main() {
 		startGin()
 	case "gocraftWeb":
 		startGocraftWeb()
+	case "goframe":
```

Comment:
@gqcn / @smallnest  for framework to be included in becnhmarks its name must be added also in list of names at  [libs.sh](https://github.com/smallnest/go-web-framework-benchmark/blob/34942a823bced1e5858f12888e9ceecac1ba349a/libs.sh#L3)

Reply:
@aldas OK, I'll commit fix.